libraries/AP_HAL/examples/RCOutput/RCOutput.cpp

```diff
@@ -31,7 +35,7 @@ void loop (void)
             hal.console->printf("reversing\n");
         }
     }
-    hal.scheduler->delay(5);
+    hal.scheduler->delay(10);
```

Comment:
The increased delay is just to slow down how fast the servos were being driven. At 6 V, driving the servos for more than a minute or two at the faster speed was causing over heats.  Other than that there was no reason so feel free to get rid of the change.